cmake/scripts/linux/ArchSetup.cmake

```diff
@@ -36,6 +36,19 @@ else()
   endif()
 endif()
 
+# temp until further cleanup is done
+# add Raspberry Pi 2 and 3 specific flags
```

Comment:
Raspberry Pi 1 is missing
If yon don't have special flags at least don't fail with `Unknown CPU: arm1176jzf-s`

Reply:
@Rechi - I left RPi1 alone where the first check would catch it.  When I inspect the `gcc -c -Q -mcpu=native --help=target` on Arch ARMv6, I see we could could adjust these to `-mtune=armv6zk ` and you're right about the fail.  Will modify, rebase, and push shortly.

Reply:
`message(SEND_ERROR "Unknown CPU: ${CPU}")` will cause to not generate the project if you want to build for Raspberry Pi 1.

Reply:
I didn't see your edit while I edited my edit :)  I was mistaken about the alternative config for RPi1 and edited my comment accordingly.  The rebased commit just removes the fail per your suggestion.  Please review and comment.

Reply:
looks good